freqtrade/strategy/interface.py

```diff
@@ -312,22 +312,30 @@ def should_sell(self, trade: Trade, rate: float, date: datetime, buy: bool,
         experimental = self.config.get('experimental', {})
 
         if buy and experimental.get('ignore_roi_if_buy_signal', False):
-            logger.debug('Buy signal still active - not selling.')
+            logger.debug(f"'{trade.pair}' - Buy signal still active. Not selling "
+                         f"(sell_type=SellType.NONE)...")
             return SellCheckTuple(sell_flag=False, sell_type=SellType.NONE)
 
         # Check if minimal roi has been reached and no longer in buy conditions (avoiding a fee)
         if self.min_roi_reached(trade=trade, current_profit=current_profit, current_time=date):
-            logger.debug('Required profit reached. Selling..')
+            logger.debug(f"'{trade.pair}' - Required profit reached. Selling "
+                         f"(sell_type=SellType.ROI)...")
             return SellCheckTuple(sell_flag=True, sell_type=SellType.ROI)
 
         if experimental.get('sell_profit_only', False):
-            logger.debug('Checking if trade is profitable..')
+            logger.debug(f"'{trade.pair}' - Checking if trade is profitable...")
```

Comment:
Well, this format was used in one of the log messages (log from develop):
```
2019-09-11 10:19:07,276 - freqtrade.persistence - DEBUG - None - current price 0.00031070, bought at 0.00031070 and calculated stop loss is at: 0.00027333 initial stop at 0.00027333. trailing stop loss saved us: 0.00000000 and max observed rate was 0.00031270
```

Aha, it was me, who introduced extra quotes, right. Actually, I made all messages in the format `Pair 'ETHBTC': message` first, but then decided that using ':' is not visible, since ':' is also used as a delimiter in the rest of the messages, and also removed 'Pair'.

Okay, your format looks fine, I'll remove the quotes (but they help grepping the messages with the pairs at the beginning)

Reply:
how about "something" else ... ? 
(grepping for Quotes will probably return a ton of false positives ...)
```
ETH/BTC - Checking if trade is profitable...
ETH/BTC -> Checking if trade is profitable...
```

(i'm no fan of quotes in outputs ...) :D

Reply:
That looks fine without the quotes, removed them everywhere...
👍